samples/previews/azure-functions/service-bus/ASBFunctions_0/AzureFunctions.ASBTrigger.Static/AzureServiceBusTriggerFunction.cs

```diff
@@ -29,11 +30,16 @@ public class AzureServiceBusTriggerFunction
         // endpoint name, and connection strings are automatically derived from FunctionName and ServiceBusTrigger attributes
         var configuration = ServiceBusTriggeredEndpointConfiguration.FromAttributes();
 
-        configuration.UseSerialization<NewtonsoftSerializer>();
```

Comment:
given this has been removed, do we need the package reference in the csproj anymore?